changeset:   55566:04ea518e5b71
parent:      55554:fa1ecb3a52df
user:        Armin Rigo <arigo@tunes.org>
date:        Mon Jun 11 10:28:26 2012 +0200
summary:     issue1164: get_python_lib(standard_lib=1) should probably just

diff -r fa1ecb3a52df -r 04ea518e5b71 lib-python/2.7/distutils/sysconfig_pypy.py
--- a/lib-python/2.7/distutils/sysconfig_pypy.py	Sun Jun 10 23:49:16 2012 +0300
+++ b/lib-python/2.7/distutils/sysconfig_pypy.py	Mon Jun 11 10:28:26 2012 +0200
@@ -39,11 +39,10 @@
     If 'prefix' is supplied, use it instead of sys.prefix or
     sys.exec_prefix -- i.e., ignore 'plat_specific'.
     """
-    if standard_lib:
-        raise DistutilsPlatformError(
-            "calls to get_python_lib(standard_lib=1) cannot succeed")
     if prefix is None:
         prefix = PREFIX
+    if standard_lib:
+        return os.path.join(prefix, "lib-python", get_python_version())
     return os.path.join(prefix, 'site-packages')
 
 

